packages/compass-import-export/src/modules/export.js

```diff
@@ -513,14 +510,6 @@ export const startExport = () => {
         fileName: exportData.fileName,
       });
       dispatch(onFinished(numDocsToExport));
-      dispatch(
-        appRegistryEmit(
-          'export-finished',
```

Comment:
Was nothing listening to this event, only the global one?

Reply:
Yep, nothing in the app, only old analytics seems to be listening to this one

Reply:
After this comment I double-checked other events that were affected by this change and did a clean-up for import-finished too in 0704881430f6f74eb9247501ddd0141a39b4a40a